build/media_source/plg_system_shortcut/shortcut.js

```diff
@@ -40,8 +40,7 @@ if (!Joomla) {
       {
         Joomla.addShortcut(e, selector);
       }
-      
-      // console.log(actionButton + " " + selector)
+    
```

Comment:
You HAVE TO rename this file to `shortcut.es6.js`, you're skipping the Code Style checks here!!!